### PR TITLE
Optimize contam tool

### PR DESCRIPTION
**Pull request by hover2pi**
Contam tool upgrades including:
- multiprocessing
- single function for all JWST instruments
- loop vectorization
- using `pysiaf` for all instrument apertures

### CODE REVIEW

**Comment by seweissman** [comment]
Any updates on getting these changes pulled in? They address performance fixes that we need for exomast's target visibility tool.

**Comment by hover2pi** [comment]
I'm not sure why this is saying there are conflicts. I pulled and resolved all of these. It's ready for review though. But don't merge this until #539 is merged!

**Review by nespinoza — APPROVED** [review]

**Comment by hover2pi** [comment]
Adding #558 to this release. Here's a reminder!

**Pull request merged**
